README.md

```diff
@@ -110,3 +110,12 @@ IMS_CLIENT_SECRET=client secret for the IMS API
 SLACK_TOKEN_WORKSPACE_EXTERNAL_ELEVATED=Slack token for the external workspace, elevated permissions
 SLACK_OPS_CHANNEL_WORKSPACE_EXTERNAL=channel ID to use for operations messages in the external workspace
 ```
+
+Import controller requires the following env variables:
+
+```plaintext
```

Comment:
with the dynamo experience we made the experience of having separate variables growing and becoming a pain. in more recent endeavors, we are trending towards having a single variable with a JSON structure inside or leveraging the spacecat global configuration dynamo table. just food for thought, nothing blocking

Reply:
Good call, will update to use a single `IMPORT_CONFIGURATION` variable.